var/spack/repos/builtin/packages/py-mikado/package.py

```diff
@@ -38,7 +38,7 @@ class PyMikado(PythonPackage):
     depends_on('py-tabulate', type=('build', 'run'))
     depends_on('py-msgpack@1.0.0:', type=('build', 'run'))
     depends_on('py-python-rapidjson@0.8:', type=('build', 'run'))
-    depends_on('py-pandas', type=('build', 'run'))
+    depends_on('py-pandas@:1.1.5', type=('build', 'run'))
```

Comment:
I would undo this change. If you're using Python 3.8, then any version of pandas should work. If you're having concretization issues, you may want to try the new concretizer, or explicitly set things like this in your `packages.yaml`.